x-pack/platform/plugins/shared/streams/server/lib/streams/client.ts

```diff
@@ -59,6 +60,7 @@ export class StreamsClient {
     private readonly dependencies: {
       scopedClusterClient: IScopedClusterClient;
       assetClient: AssetClient;
+      queryClient: QueryClient;
```

Comment:
I'm not sure if this should be exposed top-level, or if it should be part of the asset client. e.g. what happens if somebody uses the asset client to update the queries?